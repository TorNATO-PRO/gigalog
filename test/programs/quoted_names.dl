parent("John Smith", "Mary Jane").
parent("Mary Jane", "Bobby").

ancestor(X, Y) :- parent(X, Y).
ancestor(X, Y) :- parent(X, Z), ancestor(Z, Y).
