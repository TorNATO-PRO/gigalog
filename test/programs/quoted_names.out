ancestor("John Smith","Bobby")
ancestor("John Smith","Mary Jane")
ancestor("Mary Jane","Bobby")
parent("John Smith","Mary Jane")
parent("Mary Jane","Bobby")
